lib/ecto_ldap/adapter.ex

```diff
@@ -27,18 +27,25 @@ defmodule Ecto.Ldap.Adapter do
 
     {:ok, connection} = Exldap.connect
     # IEx.pry
-    {:ok, search_results} = Exldap.search_field(
+
+    search_response = Exldap.search_field(
         connection,
         prepared[:base],
         prepared[:filter_parameter],
         prepared[:filter_criteria])
-
-    IO.inspect search_results
+    
+    IO.inspect search_response
+
+    case search_response do
+      {:ok, search_results} ->
+        :ok
+      _ ->
+       search_response
+    end
```

Comment:
I'm anticipating some error-handling within this block in case there are no results.  Obviously there's nothing meaningful yet, it will be coming later.